app/src/main/java/com/github/se/gomeet/ui/authscreens/register/RegisterNameCountryPhone.kt

```diff
@@ -125,14 +204,18 @@ fun RegisterNameCountryPhone(
             keyboardOptions =
                 KeyboardOptions.Default.copy(
                     imeAction = ImeAction.Done, keyboardType = KeyboardType.Phone),
-            modifier = Modifier.fillMaxWidth())
+            modifier = Modifier.fillMaxWidth(),
+            visualTransformation = PhoneNumberVisualTransformation())
```

Comment:
I had no idea that this was a possible way of formatting the phone number, nicely done!